declarations/panorama/dota.d.ts

```diff
@@ -2303,7 +2306,8 @@ interface CPanoramaScript_VRUtils {
 
 interface DollarStatic {
     (selector: string) : Panel;
-    CreatePanel(type: string, root: Panel, name: string): Panel;
+    CreatePanel(type: string, root: Panel, id: string): Panel;
+    CreatePanel(type: string, root: Panel, id: string): PanelBase;
```

Comment:
PanelBase is not a thing? remove this line.

Reply:
PanelBase is a parent to Panel, SliderPanel and CircularProgressBar.
SliderPanel and CircularProgressBar are missing all but four Panel properties, so I moved those four into a separate interface.